lib/orderbook/MatchingEngine.ts

```diff
@@ -149,6 +151,15 @@ class MatchingEngine {
 
     return matchingResult;
   }
+
+  public removeOwnOrder = (orderId: string): orders.StampedOwnOrder | null => {
```

Comment:
Maybe we should just call this `removeOrder` and have it accept `StampedOrder` so we can reuse this same method down the road when removing orders due to peer or swap events. The logic shouldn't change whether we're removing a peer order or own order since they're all in the same queue.

Reply:
That's how it previously was, but I changed it because I figured out that soon we're gonna have to deal here with the separated lists (#215) as well, so not everything will be unified. 

In regards to having it accepting `StampedOrder`, see my other comment.